blurring_as_a_service/utils/generics.py

```diff
@@ -40,8 +40,10 @@ def delete_file(file_path):
 
 
 def copy_file(relative_path, input_path, output_path):
-    source_path = os.path.join(input_path, relative_path)
-    destination_path = os.path.join(output_path, relative_path)
+
+    # We don't use os.path.join because it doesn't work with Azure paths
+    source_path = input_path + relative_path
```

Comment:
how did it work before then?

Reply:
I don't know, but I was stuck for days because using `os.path.join()` didn't create the correct path (it was ignoring `input_path` and `output_path`). The only way I found to make it work was to concatenate the two paths with a `+`!